src/puppetlabs/trapperkeeper/internal.clj

```diff
@@ -168,15 +168,19 @@
       --debug
       --bootstrap-config <bootstrap file>
       --config <.ini file or directory>
-      --plugins <plugins directory>"
+      --plugins <plugins directory>
+      --restart-file <restart file>"
   [cli-args :- [(schema/maybe schema/Str)]]
   (let [specs       [["-d" "--debug" "Turns on debug mode"]
                      ["-b" "--bootstrap-config BOOTSTRAP-CONFIG-FILE" "Path to bootstrap config file"]
                      ["-c" "--config CONFIG-PATH"
                       (str "Path to a configuration file or directory of configuration files, "
                            "or a comma-separated list of such paths."
                            "See the documentation for a list of supported file types.")]
-                     ["-p" "--plugins PLUGINS-DIRECTORY" "Path to directory plugin .jars"]]
+                     ["-p" "--plugins PLUGINS-DIRECTORY" "Path to directory plugin .jars"]
+                     ["-r" "--restart-file RESTART-FILE"
+                      (str "Path to a file whose contents is incremented each time all of "
+                           "the configured services have been started.")]]
```

Comment:
Grammar nitpick: I think this should be "Path to a file whose contents *are* incremented," since it's the contents that are incremented.